docs/sources/reference/commandline/cli.md

```diff
@@ -302,80 +319,74 @@ Currently supported options of `devicemapper`:
     10 GB of space on the pool. However, the filesystem will use more space for
     the empty case the larger the device is.
 
-     **Warning**: This value affects the system-wide "base" empty filesystem
-     that may already be initialized and inherited by pulled images. Typically,
-     a change to this value will require additional steps to take effect:
+    **Warning**: This value affects the system-wide "base" empty filesystem
+    that may already be initialized and inherited by pulled images. Typically,
+    a change to this value will require additional steps to take effect:
 
         $ sudo service docker stop
         $ sudo rm -rf /var/lib/docker
         $ sudo service docker start
 
-    Example use:
-
-        $ docker -d --storage-opt dm.basesize=20G
+    Example use: ``$ docker -d --storage-opt dm.basesize=20G``
```

Comment:
This changes the example from a 'code block' to an inline example (here, and below).

I understand that (for maintenance), it's easier to have this the same as the man-pages, but it's inconsistent with the other examples in this document.

I'd prefer them to be consistent here. Could you change the "Example use" lines in this document back to the old formatting?